src/main/java/io/javalin/Javalin.java

```diff
@@ -84,6 +84,21 @@ public Javalin start() {
         return this;
     }
 
+
+    public Javalin ignoreTrailingSlashes(){
+        pathMatcher.setIgnoreTrailingSlashes(true);
+        return this;
+    }
+
+    public Javalin dontIgnoreTrailingSlashes(){
```

Comment:
We only need this 1 method. Javalin should ignore trailing slashes by default, and users can use this method to change that behavior. They should not be able to change it back. This method also has to be called before `start()` is called, so you need to add `ensureActionIsPerformedBeforeServerStart("Telling Javalin to not ignore slashes");`